etc/kayobe/kolla-image-tags.yml

```diff
@@ -21,3 +21,6 @@ kolla_image_tags:
   ovn_sb_db_relay:
     rocky-9: 2024.1-rocky-9-20250129T155601
     ubuntu-jammy: 2024.1-ubuntu-jammy-20250129T155601
+  horizon:
+    rocky-9: 2024.1-rocky-9-20250203T171853
+    ubuntu-jammy: 2024.1-ubuntu-jammy-20250203T171853
```

Comment:
Nit: please keep alphabetical order except for `openstack` first. This is to reduce the risk of duplicates.